sdk/storage/Azure.Storage.DataMovement/src/Shared/LocalTransferCheckpointer.cs

```diff
@@ -285,42 +289,25 @@ public override Task<List<string>> GetStoredTransfersAsync(CancellationToken can
             DataTransferStatus status,
             CancellationToken cancellationToken = default)
         {
-            long length = DataMovementConstants.OneByte * 3;
```

Comment:
were we treating an int as 3 bytes?

Reply:
No, originally, we were storing the job status as three fields (state enum, hasFailed bool, hasSkipped bool) each one byte in length. Now it's all stored in a Flags Enum backed by an int.